lib/kafo/kafo_configure.rb

```diff
@@ -371,6 +369,11 @@ def validate_all(logging = true)
 
     def run_installation
       self.class.hooking.execute(:pre)
+
+      hiera = HieraConfigurer.new(config.app[:hiera_config], config.modules, config.app[:order])
+      hiera.write_configs
```

Comment:
Everything in `#write_configs` should happen in a temporary directory - it's meant to be self-contained so we generate a set of merged Hiera configs and data files from answers from what's in memory, then we wipe it after it's used to run Puppet. No generated Hiera config or data files are kept.

Noop is still respected when saving the Kafo answers file up on lines 143-144.